resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -85,6 +85,12 @@ private[spark] object Config extends Logging {
       .stringConf
       .createOptional
 
+  val KUBERNETES_EXECUTOR_CORES =
+    ConfigBuilder("spark.kubernetes.executor.cores")
+      .doc("Specify the CPU core request for each executor pod")
```

Comment:
nit: it was lower case "cpu" in other config, and no "core"

Reply:
Done.